core/src/tpu.rs

```diff
@@ -155,6 +158,32 @@ impl Tpu {
 
         let (non_vote_sender, non_vote_receiver) = banking_tracer.create_channel_non_vote();
 
+        // Streamer for Votes:
+        let SpawnServerResult {
+            endpoints: _,
+            thread: tpu_vote_quic_t,
+            key_updater: vote_streamer_key_updater,
+        } = spawn_server_multi(
+            "solQuicTVo",
+            "quic_streamer_tpu_vote",
+            tpu_vote_quic_sockets,
+            keypair,
+            vote_packet_sender.clone(),
+            exit.clone(),
+            staked_nodes.clone(),
+            QuicServerParams {
+                max_connections_per_peer: 1,
+                max_connections_per_ipaddr_per_min: tpu_max_connections_per_ipaddr_per_minute,
+                coalesce: tpu_coalesce,
+                max_staked_connections: MAX_STAKED_CONNECTIONS
```

Comment:
I wonder if we should consider bumping up this number... I believe `MAX_STAKED_CONNECTIONS` + `MAX_UNSTAKED_CONNECTIONS` comes out to 2500 currently, but we have a little over 3k staked validators on testnet

Reply:
I checked we have 1415 staked nodes in mb.